packages/airnode-examples/scripts/run-airnode-locally.ts

```diff
@@ -23,7 +23,7 @@ const main = async () => {
   }
 
   const integrationPath = join(__dirname, '../integrations', integrationInfo.integration);
-  runShellCommand(`docker run --rm -v ${integrationPath}:/app/config --network="host" --name airnode ${imageName}`);
+  runShellCommand(`docker run --rm -it -v ${integrationPath}:/app/config --network="host" --name airnode ${imageName}`);
```

Comment:
Added this so the signals from OS are passed correctly. Without it, the container was running unless I called `docker stop`.

Reply:
I think I didn't include `-it` for a specific reason (can't remember why exactly) - but the examples tests fail currently.

Reply:
Ok, I'm giving up, spawning in the e2e test doesn't work properly with this. I'm reverting it but I'll add a notice to the examples README to stop the container manually.